app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -0,0 +1,19 @@
+.header.mb-5
+  .std-heading
+    i.fa-solid.fa-wallet
+    =< t('.header')
+
+.row
+  .col-8.col-md-4.col-lg-3
+    img.nbp_wallet_qr_code[src=nbp_wallet_qr_code_users_path(truncated_reference: @truncated_reference) data-template-validity=Enmeshed::ConnectorApi::TEMPLATE_VALIDITY_PERIOD.seconds alt=t('.qr_code_alt_text')]
+    = button_tag class: 'btn btn-primary regenerate-qr-code-button w-100 mt-3' do
+      = t('.regenerate_code')
+  .col-8.col-md-6.mt-3.ms-md-3.mt-md-0
+    p.fs-6
+      = t('.info_html', alternative_link: @template_url, app_store_link: 'https://apps.apple.com/de/app/mein-bildungsraum-wallet/id6467007352', play_store_link: 'https://play.google.com/store/apps/details?id=de.bildungsraum.wallet.beta')
```

Comment:
We had a short conversation on this comment and came up with the following:

1. Let's consider creating a `RelationshipTemplate` class, similar to the way we created a class for the `Relationship`. Then, the `RelationshipTemplate` class could contain some more responsibilities, such as returning the link or QR code (which is currently in the controller) and also return the App Store / Play Store links.
2. The two links can be moved to the settings or locales. For the locales, they could be directly included in the text (since we are referring to the Wallet app already), or in the settings under the `enmeshed` key to prevent a language-wise duplication.

Reply:
Sounds good.